CMakeLists.txt

```diff
@@ -395,7 +395,7 @@ set(RPP_RPM_PACKAGE_LIST     "rocm-hip-runtime, openmp-extras-runtime")
 set(RPP_DEBIAN_DEV_PACKAGE_LIST  "rocm-hip-runtime-dev, openmp-extras-dev, half")
 set(RPP_RPM_DEV_PACKAGE_LIST     "rocm-hip-runtime-devel, openmp-extras-devel, half")
 set(RPP_DEBIAN_TEST_PACKAGE_LIST  "python3-dev, python3-pip, python3-pandas, python3-openpyxl, libopencv-dev, libsndfile1-dev")
-set(RPP_RPM_TEST_PACKAGE_LIST     "python3-devel, python3-pip, python3-pandas, python3-openpyxl") # TBD: OpenCV & libsnd packages missing on RPM
+set(RPP_RPM_TEST_PACKAGE_LIST     "python3-devel, python3-pip") # TBD: pandas, openpyxl, OpenCV & libsnd packages missing on RPM
```

Comment:
openpyxl should be fine.
pandas is needed to display rocprof profiling results in profiling mode (performance tests without profiling should still run). ctest might be skipped - https://github.com/ROCm/rpp/blob/develop/utilities/test_suite/CMakeLists.txt#L263-L265